### PR TITLE
1.3.4 release notes

### PR DESCRIPTION
**Pull request by sdake**

### CODE REVIEW

File: content/en/news/2019/announcing-1.3.4/index.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+aliases:
+    - /news/announcing-1.3.4
+---
+
+This release includes bug fixes to improve robustness. This release note describes what's different between Istio 1.3.3 and Istio 1.3.4.
+
+{{< relnote >}}
+
+## Bug fixes
+- **Fixed** a nil dereference in the Google nodeagent provider. ([Pull Request #18296](https://github.com/istio/istio/pull/18260))
```
Comment (geeknoid) [inline content/en/news/2019/announcing-1.3.4/index.md]:
```suggestion
- **Fixed** a crashing bug in the Google nodeagent provider. ([Pull Request #18296](https://github.com/istio/istio/pull/18260))
```

**Comment by sdake** [comment]
Rendering of notes: https://5dbcdcd521889200083fe2b0--preliminary-istio.netlify.com/news/2019/announcing-1.3.4/

File: content/en/news/2019/announcing-1.3.4/index.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+- **Fixed** Prometheus annotations and updated Jaeger to 1.14. ([Pull Request #18274](https://github.com/istio/istio/pull/18274))
+- **Fixed** in-bound listener reloads that occur on 5 minute intervals. ([Issue #18138](https://github.com/istio/istio/issues/18088))
+- **Fixed** validation of key and certificate rotation. ([Issue #17718](https://github.com/istio/istio/issues/17718))
+- **Fixed** invalid internal resource garbage collection. ([Issue #16818](https://github.com/istio/istio/issues/16818))
+- **Fixed** webhooks that were not updated on a failure. ([Pull Request #17820](https://github.com/istio/istio/pull/17820)
+- **Improved** performance of OpenCensus tracing adapter. ([Issue #18042](https://github.com/istio/istio/issues/18042))
```
Comment (geeknoid) [inline content/en/news/2019/announcing-1.3.4/index.md]: Add

## Minor enhancements

File: content/en/news/2019/announcing-1.3.4/index.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+- **Improved** reliability of SDS service. ([Issue #17409](https://github.com/istio/istio/issues/17409), [Issue #17905](https://github.com/istio/istio/issues/17905])
```
Comment (geeknoid) [inline content/en/news/2019/announcing-1.3.4/index.md]:
```suggestion
- **Improved** reliability of the SDS service. ([Issue #17409](https://github.com/istio/istio/issues/17409), [Issue #17905](https://github.com/istio/istio/issues/17905])
```

File: content/en/news/2019/announcing-1.3.4/index.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+---
+title: Announcing Istio 1.3.4
+description: Istio 1.3.3 release announcement.
```
Comment (brian-avery) [inline content/en/news/2019/announcing-1.3.4/index.md]: this should be 1.3.4

File: content/en/news/2019/announcing-1.3.4/index.md (hunk 3 above)
Comment (sdake) [reply content/en/news/2019/announcing-1.3.4/index.md]: addressed.

File: content/en/news/2019/announcing-1.3.4/index.md (hunk 2 above)
Comment (sdake) [reply content/en/news/2019/announcing-1.3.4/index.md]: addded.

File: content/en/news/2019/announcing-1.3.4/index.md (hunk 1 above)
Comment (sdake) [reply content/en/news/2019/announcing-1.3.4/index.md]: addressed

File: content/en/news/2019/announcing-1.3.4/index.md (hunk 4 above)
Comment (sdake) [reply content/en/news/2019/announcing-1.3.4/index.md]: addressed

**Comment by sdake** [comment]
@geeknoid I wasn't sure where this pr should be cherrypicked to.  We plan to release ASAP - so the sooner this hits the live site, the better.

Cheers
-steve

**Pull request merged**